src/main/java/frc/robot/commands/WristOpenCommand.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Arm;
+import frc.robot.subsystems.Manipulator;
+
+public class WristOpenCommand extends CommandBase {
```

Comment:
look into InstantCommand, and make into a Command factory method of Manipulator - see 2022's subsystems for examples

Reply:
I'll also commit what I'm looking for this